.azure/pipelines/components-e2e-tests.yml

```
@@ -44,9 +44,9 @@ jobs:
       displayName: NPM install
     - script: .dotnet/dotnet build ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-restore
       displayName: Build
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter QuarantinedTest\!~github --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined!=true|Quarantined=false' --logger trx
```

While here, let's clean up the results file names and add HTML results. Suggestion indentation looks a bit odd however.

```suggestion
    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined!=true|Quarantined=false'
                 --logger:"trx%3BLogFileName=Microsoft.AspNetCore.Components.E2ETests.trx"
                 --logger:"html%3BLogFileName=Microsoft.AspNetCore.Components.E2ETests.html"
                 "--ResultsDirectory:$(Build.SourcesDirectory)/artifacts/TestResults/$(_BuildConfig)"
```